app/services/api.js

```diff
@@ -0,0 +1,54 @@
+import axios from 'axios';
+import qs from 'qs';
+import Alert from 'react-s-alert';
+import currentUser from 'services/currentUser';
+import sessionStorage from 'services/sessionStorage';
+import i18n from 'services/i18n';
+import config from 'config';
+
+const api = axios.create({
+  baseURL: config.apiTarget,
+  paramsSerializer: qs.stringify
```

Comment:
Why we can't use default behavior? `qs` is used inside Axios.

Reply:
Good point, thank you. By default it actually uses `qs.stringify(params, { arrayFormat: 'brackets' })`. But it works for us. Will change it in next commit.